python/CMakeLists.txt

```
@@ -59,6 +51,7 @@ if(NOT RMM_FOUND)
   add_subdirectory(../ rmm-cpp)
 endif()
 
-add_subdirectory(cmake)
+include(rapids-cython)
+rapids_cython_init()
```

This seems like it stands alone. A newline between might be helpful.
```suggestion
rapids_cython_init()

```